src/api/app/models/configuration.rb

```diff
@@ -71,6 +71,19 @@ def ldapgroup_enabled?
   end
   # End of class methods
 
+  def ldap_enabled?
+    CONFIG['ldap_mode'] == :on
+  end
+
+  def passwords_changable?
+    change_password && CONFIG['proxy_auth_mode'] != :on && CONFIG['ldap_mode'] != :on
+  end
+
+  def accounts_editable?
+    (CONFIG['proxy_auth_mode'] != :on || (CONFIG['proxy_auth_mode'] == :on && !CONFIG['proxy_auth_account_page'].blank?)) && \
+      CONFIG['ldap_mode'] != :on
+  end
+
```

Comment:
[nitpick] General advice: Reading conditions like you wrote it here is hard to understand as you do it always the other way around (``!= on`` vs. ``== :off``, ``!CONFIG['proxy_auth_account_page'].blank?`` vs ``CONFIG['proxy_auth_account_page'].present?``

See also the other methods you introduced. I don't require that you change it but I think it is in general a good practice (rubocop usually also complains about that).